pkg/config/config.go

```diff
@@ -679,6 +685,23 @@ func load(config Config, origin string, loadSecret bool) error {
 		}
 	}
 
+	// If this variable is set to true, we'll use DefaultPython for the Python version,
+	// ignoring the python_version configuration value.
+	if ForceDefaultPython == "true" {
+		defaultPython, err := strconv.Atoi(DefaultPython)
+		if err != nil {
+			log.Warn("Bad value for DefaultPython, will use configuration value. Err:", err)
+		}
+		override := make(map[string]interface{})
+		override["python_version"] = defaultPython
```

Comment:
is it OK to override a config value with a different type than its default value?

i.e. here the config value is set to a string by default (https://github.com/DataDog/datadog-agent/blob/45fea0f26b3b4ead3ccab644d53282ef8f9d34b1/pkg/config/config.go#L144), and you override it with an int here

Reply:
I'll use `DefaultPython` directly as a string, resolving both comments in one shot. 👍